docs/pages/versions/unversioned/introduction/testing-with-jest.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Testing with Jest
+---
+
+Adding Jest to your Expo app has gotten pretty simple, to help with this process, we've even created a [Jest preset](https://github.com/expo/expo/tree/master/packages/jest-expo) for you. Let's get started.
+
+## Installation
+
+First we need to install jest-expo: `yarn add jest-expo --dev` **or** `npm i jest-expo --save-dev`
+
+Then we need to add/update `package.json` to include:
+
+```js
+"scripts": {
+  ...
+  "test": "jest"
+},
+"jest": {
+  "preset": "jest-expo"
+}
+```
+
+Now let's add [enzyme](https://airbnb.io/enzyme/), [enzyme adapter for react < 16](https://airbnb.io/enzyme/#upgrading-from-enzyme-2x-or-react--16) and [react-dom](https://github.com/facebook/react/tree/master/packages/react-dom):
+
+`yarn add react-dom enzyme enzyme-adapter-react-16 --dev` **or** `npm i react-dom enzyme enzyme-adapter-react-16 --save-dev`
+
+With enzyme's adapter for react < 16, we need to have this set up before every test that runs. Jest gives us a way to do that, head back over to `package.json`:
+
+```js
+"jest": {
+  "preset": "jest-expo",
+  "setupFiles": [
+    "<rootDir>/jest.setup.js"
+  ]
+}
+```
+
+Now create the file `jest.setup.js`
+
+```js
+import { configure } from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+```
+
+That's it! Now we can start writing Jest tests!
+
+## My First Jest Test
```

Comment:
we should indicate that this is a snapshot test and explain what that is and why its useful